examples/old/cub200/upload_cub2010.py

```diff
@@ -38,8 +38,8 @@ def forward(self, image_info):
         ds["seg"] = np.empty(1, object)
         ds["seg"][0] = image_info["anno"]["seg"]
 
-        ds["bbox"] = np.empty(1, object)
-        ds["bbox"][0] = image_info["anno"]["bbox"]
+        ds[""] = np.empty(1, object)
```

Comment:
@thisiseshan, is this change related to this PR?

Reply:
No, this change is not part of this PR. Seems like when I was going through the examples I accidentally deleted the 2 lines.
Please revert it back to this:

```suggestion
        ds["bbox"] = np.empty(1, object)
        ds["bbox"][0] = image_info["anno"]["bbox"]
```